docs/user-guide/utils.rst

```diff
@@ -32,3 +102,33 @@ Splitting a mixed file or stream of bytes by APID so they can be used with the `
    Writing ./apid00385.tlm...
```

Comment:
everywhere else you use `.bin` for packet files.

Reply:
Yeah, but this feature isn't new so if we change it, it would break backwards incompatibility.  I changed all the other instances to `.tlm`